### ISSUE TITLE
Migrate sample apps to "auto-build off"

### ISSUE THREAD

**Issue by chefsalim** [habitat-sh/builder#587]
Now that we have auto-build capability, we need to plan and execute a migration of existing sample applications to have auto build turned off.  

This should be straightforward from a technical standpoint once the shard merge migration is complete. 

The migration should be communicated to the community appropriately.  This can be via Forum + Slack posts at a minimum, but could also include individual communication to owners of packages impacted.

**Comment by chefsalim** [comment]
Duplicate of #571
